docs/source/contributing/prerequisites.md

```diff
@@ -1,4 +1,8 @@
-# Prerequisites
+---
+title: "Prerequisites"
+sidebar_title: "System Prerequisites"
+description: "Before contributing..."
+---
 
 The Apollo CLI tool is written in [Rust]. In order to contirbute, you'll need to have
 Rust installed. To install Rust, visit [https://www.rust-lang.org/tools/install].
```

Comment:
we probably need to add here that if you want to contribute docs you need JavaScript. @jbaxleyiii i think we should take a small moment to evaluate if we are ok with the dependency (i personally am, but it's true that we have non JS users who now need to install JS to contribute when they previously did not.)